migration/idempotent/010-telemetry.sql

```diff
@@ -115,6 +115,9 @@ $$
 
         SELECT public.approximate_row_count('_ps_trace.span')::TEXT INTO result;
         PERFORM _ps_catalog.apply_telemetry('traces_spans_total_approx', result);
+        -- Trace spans processed by jaeger collector will always have an internal attribute named 'internal.span.format' with possible values 'jaeger|zipkin|proto' according to https://github.com/jaegertracing/jaeger/issues/1490. We can use this to infer whether promscale's gRPC remote storage implementation has been used or not.
+        SELECT count(*)::TEXT INTO result FROM _ps_trace.tag WHERE key='internal.span.format';
+        PERFORM _ps_catalog.apply_telemetry('traces_jaeger_span_ingested', result);
```

Comment:
Sorry, I didn't understand why are we doing `count(*)` here (unless we want to count types of ingestion, i.e., jaeger/zipkin/proto). Shouldn't this field be a boolean, like `count(*) > 0`?

Reply:
Yes, the intention is to count the number of span types.

Reply:
May be we can create one metric per type for a better clarity?

```
FOR type IN (SELECT value FROM _ps_trace.tag WHERE key='internal.span.format')
LOOP
PERFORM _ps_catalog.apply_telemetry('traces_jaeger_span_type_' || type, '1');
END LOOP;
```

Reply:
Another simple approach is aggregating as string array,

```
SELECT array_agg(value) FROM _ps_trace.tag WHERE key='internal.span.format';
```

Reply:
I like the array_agg value. I think, we should also change the name of telemetry, to something like `traces_jaeger_span_ingested_types` which suits more to its value, which is agg form of types of Jaeger traces.

Then, it should be good to merge.

Reply:
I named the event as `promscale_traces_jaeger_span_types`. Hope that is fine.